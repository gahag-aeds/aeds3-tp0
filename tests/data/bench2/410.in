410 205
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 
Sub 166 301
Min 160 295
Sub 120 255
Max 138 273
Sub 36 171
Sum 79 214
Sub 178 313
Min 106 241
Add 19 154
Sum 90 225
Add 125 260
Max 105 240
Sub 124 259
Max 118 253
Add 43 178
Min 132 267
Add 65 200
Max 165 300
Add 175 310
Max 215 350
Add 5 140
Min 155 290
Add 137 272
Max 133 268
Add 245 380
Min 35 170
Add 121 256
Min 62 197
Sub 82 217
Sum 226 361
Sub 86 221
Sum 248 383
Add 111 246
Sum 205 340
Add 231 366
Max 147 282
Add 129 264
Min 134 269
Sub 98 233
Min 147 282
Add 69 204
Sum 222 357
Add 97 232
Max 192 327
Sub 186 321
Max 259 394
Add 49 184
Sum 250 385
Add 269 404
Max 70 205
Sub 36 171
Min 274 409
Sub 224 359
Max 18 153
Sub 132 267
Sum 41 176
Sub 52 187
Max 98 233
Sub 102 237
Max 133 268
Sub 170 305
Sum 188 323
Sub 226 361
Max 6 141
Sub 238 373
Sum 182 317
Sub 272 407
Sum 93 228
Add 41 176
Sum 95 230
Add 85 220
Min 110 245
Add 163 298
Max 28 163
Add 147 282
Sum 74 209
Sub 132 267
Max 196 331
Add 169 304
Max 127 262
Add 111 246
Min 204 339
Sub 126 261
Sum 180 315
Add 221 356
Min 103 238
Sub 220 355
Min 118 253
Sub 46 181
Max 48 183
Sub 96 231
Max 215 350
Add 81 216
Max 48 183
Add 221 356
Min 44 179
Add 229 364
Max 64 199
Sub 256 391
Max 116 251
Add 5 140
Sum 187 322
Add 71 206
Sum 167 302
Sub 60 195
Max 217 352
Sub 86 221
Max 191 326
Sub 258 393
Max 254 389
Add 43 178
Max 214 349
Add 183 318
Max 14 149
Sub 120 255
Min 249 384
Sub 116 251
Sum 65 200
Sub 92 227
Max 162 297
Sub 232 367
Sum 187 322
Sub 102 237
Sum 38 173
Sub 80 215
Min 168 303
Sub 202 337
Min 155 290
Sub 78 213
Sum 29 164
Sub 270 405
Sum 82 217
Add 61 196
Max 186 321
Sub 94 229
Min 120 255
Sub 248 383
Max 179 314
Add 37 172
Sum 78 213
Sub 4 139
Sum 199 334
Add 17 152
Max 186 321
Add 213 348
Max 136 271
Sub 6 141
Sum 54 189
Add 201 336
Sum 97 232
Add 61 196
Sum 158 293
Add 129 264
Sum 9 144
Sub 42 177
Max 209 344
Sub 22 157
Sum 89 224
Add 89 224
Min 99 234
Add 117 252
Min 204 339
Sub 26 161
Max 178 313
Add 235 370
Sum 119 254
Add 143 278
Min 54 189
Add 143 278
Max 179 314
Add 131 266
Max 146 281
Sub 224 359
Min 148 283
Add 177 312
Min 162 297
Add 9 144
Sum 182 317
Add 61 196
Min 55 190
Sub 124 259
Sum 122 257
Add 59 194
Sum 253 388
Sub 250 385
Max 100 235
Add 33 168
Sum 271 406
Sub 188 323
Sum 121 256
Sub 96 231
Sum 150 285
Sub 170 305
Max 241 376
Add 173 308
Min 250 385
Sub 86 221
Max 42 177
Add 29 164
Sum 228 363
Sub 66 201
Min 160 295
Sub 100 235
Sum 15 150
Add 33 168
