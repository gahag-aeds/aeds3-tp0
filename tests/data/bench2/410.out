-1
97
6577
-3
6676
97
96
-3
98
99
-1
99
2
1
6910
6805
6849
100
1
0
7134
101
101
6889
104
1
103
6938
99
98
6952
102
6778
6739
6902
0
103
7160
98
101
-3
6985
2
2
102
98
102
1
102
100
7137
7126
99
99
98
98
104
-2
6999
98
6914
6741
-5
-5
6430
6393
96
-6
95
6490
6097
95
96
6449
6346
6612
6795
98
6459
-3
-4
98
6900
-1
101
102
-1
-2
7333
3
7287
6404
102
6315
7387
7145
98
-5
106
6507
-4
6751
